code/modules/reagents/chemistry/reagents/toxin_reagents.dm

```diff
@@ -70,16 +73,6 @@
 		C.adjustPlasma(20)
 	return ..()
 
-/datum/reagent/toxin/plasma/reaction_obj(obj/O, reac_volume)
```

Comment:
this is one of those "people complaining about near unused feature that only popped up with people griefing" features